client/db_test.go

```diff
@@ -266,7 +266,7 @@ func ExampleTx_Commit() {
 	// 1/0: ab=2
 }
 
-func ExampleDB_Insecure() {
+func ExampleDB_second() {
```

Comment:
I think `_insecure` is fine. That shows up as `Example (Insecure)` in godoc, and it looks like the example is mostly checking the insecure flag, so that seems to make sense.